otherlibs/raw_spacetime_lib/spacetime_offline.c

```diff
@@ -83,7 +83,6 @@ CAMLprim value caml_spacetime_unmarshal_trie (value v_channel)
 
 CAMLprim value caml_spacetime_node_num_header_words(value unit)
 {
-  unit = Val_unit;
```

Comment:
Isn't
```c
  (void)unit;
```
the usual way to bypass unused warnings?

Reply:
We don't try to bypass "unused argument" warnings, because 1- we have a lot of functions with legitimately-unused arguments, 2- we don't activate the "unused argument" warning, and 3- if we were to activate it, I'd rather mark unused arguments using GCC/Clang attributes.